src/components/CalendarMonthGrid.jsx

```diff
@@ -303,6 +353,15 @@ class CalendarMonthGrid extends React.PureComponent {
         ref={this.setContainerRef}
         onTransitionEnd={onMonthTransitionEnd}
       >
+        <div id={dayLabelledbyId} hidden>
```

Comment:
I don't think that this approach will work - by default in VO on iOS, when you navigate to different elements on the page, it doesn't move keyboard focus. This would mean that this text will not change as you move through the different days and keep saying that the same day.

Reply:
If you're willing to create new phrases, you can do a mixture of the approach you have here and what I had suggested with using two references within `aria-labelledby`. That way this container updates based on the qualification (checkout/checkin), and the id references don't change, but the date strings are tied to the date itself and not this node.

Reply:
rats! It sounds like we might need to break this into two parts then, with one part of the phrase here and the other part of the phrase in the CalendarDay?

I'm not sure we can make that big of a breaking change here. @nkinser what do you think?

Reply:
For reference, here's the pseudo-code you shared with me elsewhere that illustrates what we are talking about here:
```
<div id="endDatePhrase" hidden>Select as your end date</div>
...
<button aria-labelledby="05292019 endDatePhrase">
  <div id="05292019" hidden>Wednesday, May 29, 2019</div>
  29
</button>
```

Reply:
Hmm, this is tricky. Phrases can be functions that take arguments like the day, and I think some people are customizing these voiceover phrases to provide more specific information about individual days than what we currently support out of the box. I think I'll need to work through this with @nkinser to make sure we have all of our use cases properly covered.

Reply:
Sorry for the delayed response. 
I think this would make it impossible to customize the phrase based on the state of the calendar, which is really important for the pdp calendar since there's so much information that we display visually that we need to also convey to VO users. I think this is a really complicated issue so it's probably a good idea to talk about this in person